corporate/lib/stripe.py

```diff
@@ -1948,9 +2055,14 @@ def get_billing_context_from_plan(
         else:  # nocoverage
             raise BillingError(f"stripe_customer_id is None for {customer}")
 
-        fixed_price = (
-            cents_to_dollar_string(plan.fixed_price) if plan.fixed_price is not None else None
-        )
+        renewal_amount = cents_to_dollar_string(renewal_cents)
+        fixed_price: Optional[float] = None
+        if plan.fixed_price is not None:
+            fixed_price = plan.fixed_price
+            if plan.billing_schedule == CustomerPlan.BILLING_SCHEDULE_MONTHLY:
+                fixed_price = plan.fixed_price / 12
+            renewal_amount = format_money(fixed_price)
```

Comment:
Should this use `get_amount_due_fixed_price_plan`?

Reply:
No, that returns the amount in integer to be used as invoice amount.

Reply:
The correct fix was to update 
```python
    def get_customer_plan_renewal_amount(
        self,
        plan: CustomerPlan,
        last_ledger_entry: LicenseLedger,
    ) -> int:
        if plan.fixed_price is not None:
            return get_amount_due_fixed_price_plan(plan.fixed_price, plan.billing_schedule)
        ...
```

This code block was not required at all. cleaned this up.